internal/host/host.go

```diff
@@ -309,15 +309,22 @@ func (m *Manager) UpdateInventory(ctx context.Context, h *models.Host, inventory
 }
 
 func (m *Manager) RefreshStatus(ctx context.Context, h *models.Host, db *gorm.DB) error {
+	log := logutil.FromContext(ctx, m.log)
```

Comment:
Only "handle" an error once. 
Logging counts as handling.
Just add logging in case of error where we call  RefreshStatus if there isn't one.

Reply:
Fixed.